docs/source/glossary.rst

```diff
@@ -0,0 +1,131 @@
+      Often confused with :term:`prediction intervals <forecast prediction intervals>`,
+      see `The difference between prediction intervals and confidence intervals <https://robjhyndman.com/hyndsight/intervals/>`_ to understand the difference.
+
+   Forecast prediction intervals
+      Prediction intervals for predicted random variables.
+      Set of intervals for every point in the :term:`horizon <forecasting horizon>` can be called a prediction band.
+      Often confused with :term:`confidence intervals <forecast confidence intervals>`,
+      see `The difference between prediction intervals and confidence intervals <https://robjhyndman.com/hyndsight/intervals/>`_ to understand the difference.
+
+   Forecast prediction components
+      In forecast decomposition each point is represented as the sum or product of some fixed terms. These terms are called components.
+
+   Backtesting
+      Type of cross-validation when we check the quality of the forecast model using historical data.
+
+   Per-segment / local approach
+      Mode of operation when there is a separate :term:`model` / :term:`transform` for each :term:`segment` of the dataset.
+
+   Multi-segment / Global approach
+      Mode of operation when there is one :term:`model` / :term:`transform` for every :term:`segment` of the dataset.
+
+   Forecasting strategy
+      Algorithm for using an ML model to produce a multi-step time series :term:`forecast <forecasting>`.
+      See :doc:`tutorials/09-forecasting_strategies`.
+
+   Clustering
+      The task of finding clusters of similar time series.
+
+   Classification
+      The task of predicting a categorical label for the whole time series.
+
+   Segmentation
+      The task of dividing each time series into sequence of intervals with different characteristics.
+      These intervals are separated by :term:`change-points <change-point>`.
+      This shouldn't be confused with the term :term:`segment`.
+
+   Dataset
+      Collection of time series to work with.
+      In the context of the library this is often used to refer to :py:class:`~etna.datasets.tsdataset.TSDataset`.
+
+   Model
+      Entity for learning time series patterns to make a :term:`forecast <forecasting>`. See :doc:`api_reference/models`.
+
+   Transform
+      Entity for performing transformations on a :term:`dataset`. See :doc:`api_reference/transforms`.
+
+   Pipeline
+      High-level entity for solving :term:`forecasting` task. Works with :term:`dataset`, :term:`model`, :term:`transforms <transform>` and other :term:`pipelines <pipeline>`.
+
+   Lags
+      The features generated by :py:class:`~etna.transforms.math.lags.LagTransform`.
+
+   Date flags
+      The features generated by :py:class:`~etna.transforms.timestamp.date_flags.DateFlagsTransform`.
+
+   Fourier terms
+      The features generated by :py:class:`~etna.transforms.timestamp.fourier.FourierTransform`.
```

Comment:
Should we add a bit more detailed definition? Right now it is not very clear what these values are.

Reply:
Users could read it in the documentation of `FourierTransform` I suppose.